src/DB_Command.php

```
@@ -835,6 +843,26 @@ public function export( $args, $assoc_args ) {
 			}
 		}
 
+		$exclude_tables_data = Utils\get_flag_value( $assoc_args, 'exclude_tables_data', '' );
+		if ( ! empty( $exclude_tables_data ) ) {
+			unset( $assoc_args['exclude_tables_data'] );
+
+			if ( 'mariadb' !== Utils\get_db_type() ) {
+				WP_CLI::error( 'The --exclude_tables_data option is only supported by MariaDB.' );
+			}
+
+			$tables = explode( ',', trim( $exclude_tables_data, ',' ) );
```

PHPStan (2.2.7)

[error] Parameter `#1` $string of function trim expects string, mixed given.


(argument.type)